### PR TITLE
feat(module:theme): adding Korean locale

### PR DESCRIPTION
**Pull request by revfactory**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [v] The commit message follows our guidelines: https://github.com/ng-alain/delon/blob/master/CONTRIBUTING.md#commit
- [v] Tests for the changes have been added (for bug fixes / features)
- [v] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[v] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Application (the showcase website) / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: #485

## Does this PR introduce a breaking change?
```
[ ] Yes
[v] No
```

### CODE REVIEW

File: packages/schematics/docs/plugin.zh-CN.md [hunk 1]
```diff
@@ -110,6 +110,7 @@ ng g ng-alain:plugin defaultLanguage --defaultLanguage=zh-tw
 | 土耳其语       | tr     | tr        | tr_TR            | tr_TR       |
 | 波兰语    | pl     | pl        | pl_PL            | pl_PL       |
 | 希腊语    | el     | el        | el_GR            | el_GR       |
+| 朝鲜的    | ko     | ko        | ko_KR            | ko_KR       |
```
Comment (cipchk) [inline packages/schematics/docs/plugin.zh-CN.md]:
```suggestion
| 朝鲜语    | ko     | ko        | ko_KR            | ko_KR       |
```

File: packages/theme/docs/locale.zh-CN.md [hunk 2]
```diff
@@ -48,6 +48,7 @@ switchLanguage() {
 | 土耳其语     | tr_TR      |
 | 波兰语       | pl_PL      |
 | 希腊语       | el_GR      |
+| 朝鲜的       | ko_KR      |
```
Comment (cipchk) [inline packages/theme/docs/locale.zh-CN.md]:
```suggestion
| 朝鲜语       | ko_KR      |
```

**Comment by revfactory** [comment]
@cipchk Applied to PR review code. Thanks

**Comment by cipchk** [comment]
@revfactory hi, pls try merge master, i found that travis can't continue.

**Pull request merged**